docs/src/language_servers.md

```diff
@@ -717,6 +717,11 @@ Follow installation instructions on [LSP-tailwindcss](https://github.com/sublime
 
 Follow installation instructions on [LSP-terraform](https://github.com/sublimelsp/LSP-terraform).
 
+## Toit
+
+Install the [Toit](https://packagecontrol.io/packages/Toit) package from Package Control for syntax highlighting.
+Follow installation instructions on [LSP-toit](https://github.com/toitlang/toit/tree/master/tools/lsp/server).
+
```

Comment:
you are missing the 3rd step, and that is configuration that a user needs to setup. Look at other examples in the language_servers.md, it usually stays with:

3. Open `Preferences > Package Settings > LSP > Settings` and add the `"typst-lsp"` client configuration to the `"clients"`:
```
// The confirmation 
```